gpsd_client/src/client.cpp

```diff
@@ -18,7 +21,11 @@ namespace gpsd_client
       use_gps_time_(true),
       check_fix_by_variance_(true),
       frame_id_("gps")
-    {}
+    {
+      timer_ = create_wall_timer(1s, std::bind(&GPSDClientComponent::step, this));
+      start();
+      RCLCPP_INFO(this->get_logger(), "Instantiated.");
+    }
 
     bool start()
```

Comment:
This is only vaguely related to this pull request, however:

`start()`, `step()` and `stop()` should probably be private methods. The first two are only called internally in the node/component after this pull request gets merged, and the latter isn't actually used. I don't think there's a way to actually interface with them/call them externally since this code is compiled as a component. Long term they should probably be replaced by LifecycleNode functionality, but this is dependent on https://github.com/ros2/design/pull/272.